### DISCUSSION TITLE
Proxy Issue

### DISCUSSION THREAD

**Discussion by juliemars** [haugene/docker-transmission-openvpn#2614] [General]
Hi all

Before I open an issue, has anything changed with the proxy included in the docker image?

Using the Dev version pushed yesterday I get the below:

`Privoxy: Starting
Privoxy: Using config file at /etc/privoxy/config
Privoxy: Setting port to 8888
Privoxy: ERROR. Did not start correctly, outputting logs
7f5906f9cb80 Fatal error: can't bind to 172.20.0.2:8888: There may be another Privoxy or some other proxy running on port 8888`

using a older build like 5 works fine. 

Thanks

**Comment by HornetNL** [comment]
Same here.

Thanks for the tip to go back to the 5.0.0 build, that works for me as well.

**Comment by pkishino** [comment]
yes..see here #2692
